### PR TITLE
Added standard JS code linting and refactor code

### LINKED ISSUE #109: Apply js-standard code guidelines on eslint
Currently we are not using a standard.
Between the options, https://standardjs.com/ is the most advanced and no bureaucracy.

### PR DESCRIPTION
**Pull request by kdhttps**
close #109 

1. Added Standard JS Code listing and formatting. Zero configurations, more details https://standardjs.com/ 
2. Added one more feature to check linting before commit in local machine more details here https://www.npmjs.com/package/husky

### CODE REVIEW

File: test/app.init.test.js [hunk 1]
```diff
@@ -13,210 +13,199 @@ const file = chaiFiles.file
...
+        'spTLSKey'
+      )
+    })
+
+    it('server uri should be same from basic configuration file', () => {
+      expect(response.body.conf.serverURI).to.be.equal(gluuUrl)
+    })
+
+    it('expect spTLSCert file to exist', () => {
+      // eslint-disable-next-line no-unused-expressions
```
Comment (christian-hawk) [inline test/app.init.test.js:202]: Hey @kdhttps What if we use https://www.npmjs.com/package/eslint-plugin-chai-friendly instead of the comments?

**Review by christian-hawk — CHANGES_REQUESTED** [review]
Thanks for this @kdhttps ! 
Please check my minor change request feel free to discuss it.

File: test/app.init.test.js (hunk 1 above)
Comment (kdhttps) [reply test/app.init.test.js:202]: I tried but standard-js has no rules overwrite configuration facility. I guess we are trying to configuring this globally so we can configure it file_globally for specific file by adding `eslint-disable-next-line no-unused-expressions` flag at the top of the file. @christian-hawk what you think?

**Comment by kdhttps** [comment]
@christian-hawk so I've added [standardx](https://github.com/standard/standardx) by which we can use standard js feature as well as overwrite some custom rules so we have both features now, also fixed the `no-unused-expressions`. Please review

**Review by christian-hawk — APPROVED** [review]
Approved, nice job.

**Pull request merged**